global/lang/en_GB/home.json

```diff
@@ -26,7 +26,6 @@
     "team.role.megamix": "Osekai Team Organization Leader & Community Manager",
     "team.role.mulraf": "Project Lead & Lead Developer",
     "team.role.tanza": "Project Lead, Developer & Lead Designer",
-    "team.role.monoxmellow": "Video Editor & Motion Graphic Designer",

```

Comment:
rename this, we need to reuse it for our new editor, just rename it to team.role.generic.videoeditor or something